src/Tokenizer.js

```diff
@@ -29,6 +29,38 @@ function outputLink(cap, link, raw) {
   }
 }
 
+function indentCodeCompensation(raw, text) {
+  if (raw === undefined) {
+    return text;
+  }
```

Comment:
`raw` should always be a string. I think if it is `undefined` it should throw an error on the `.match` call below instead of returning `text`. It would be easier to debug.
```suggestion
```